identity-wallet/src/state/reducers/dev_mode.rs

```diff
@@ -251,5 +260,11 @@ async fn load_credential_images(
     let file_name = format!("{}.png", badge_id);
     let mut file = File::create(ASSETS_DIR.lock().unwrap().as_path().to_owned().join(file_name))?;
     copy(&mut image_bytes, &mut file)?;
+
+    let mut image_bytes: &[u8] = include_bytes!("../../../resources/images/edubadge-1.png");
+    let file_name = format!("{}.png", edubadge_id);
+    let mut file = File::create(ASSETS_DIR.lock().unwrap().as_path().to_owned().join(file_name))?;
```

Comment:
Why not use the file function to write (write_all)? https://doc.rust-lang.org/std/fs/struct.File.html

Reply:
I extracted a helper function and switched to `file.write_all()` 👍